### PR TITLE
Adding event-specific calendar icon and location preview to post-preview organism

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
This adds any event-specific styles to the post-preview organism

## Testing
- Visit http://localhost:8000/browse-filterable/

## Review
- @jimmynotjim 
- @anselmbradford 
- @sebworks 

## Preview

image

image

### CODE REVIEW

File: cfgov/preprocessed/css/organisms/post-preview.less [hunk 1]
```diff
@@ -102,6 +102,17 @@
         .grid_column(8);
     });
 }
+
+.o-post-preview_image-container {
+    position: relative;
+}
+
+.o-post-preview_image-container .calendar-icon {
```
Comment (anselmbradford) [inline cfgov/preprocessed/css/organisms/post-preview.less]: Nest this into L:108 so as not to repeat `.o-post-preview_image-container`?

File: cfgov/jinja2/v1/_includes/organisms/post-preview.html [hunk 2]
```diff
@@ -60,7 +65,26 @@
...
+            <img class="o-post-preview_image"
+                 src="{{ location_image_url({
+                            'location': settings.event_city
+                                        | default('Washington, DC', true)
+                                        | urlencode,
+                            'zoom':     '12',
+                            'scale':    '2',
+                            'size':     '276x155'
+                        }) }}"
+                 alt="Google Maps image of {{ settings.event_venue }}">
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: Won't some events be online and not have an associated map? This is probably carryover from the existing codebase, but just checking.

Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: Yes, you're right. Standby and I'll add that in. Thanks!

File: cfgov/jinja2/v1/_includes/organisms/post-preview.html [hunk 3]
```diff
@@ -71,9 +95,9 @@
         <h3 class="o-post-preview_title">
             {{ settings.heading | safe }}
         </h3>
-        {% if settings.event_location %}
+        {% if settings.event_start_dt %}
             <span class="o-post-preview_subtitle h6">
-            {{settings.event_location}} - {{ time.render(settings.event_date) }}
+            {{ settings.event_city }}, {{ settings.state }} {{ settings.event_venue if settings.event_venue else '' }} - {{ time.render(settings.event_start_dt) }}
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: Add a line break here

Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: A line break for each thing being pulled in?

Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: I don't know, but it's hard to review due to the horizontal scrolling so it'd be nice to break it up a bit

Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: Sure. I just wanted to figure out what the intent was. Will do.

File: cfgov/preprocessed/css/organisms/post-preview.less (hunk 1 above)
Comment (KimberlyMunoz) [inline cfgov/preprocessed/css/organisms/post-preview.less]: Fixed in 555ef06

File: cfgov/jinja2/v1/_includes/organisms/post-preview.html (hunk 2 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: Added with a placeholder image in 555ef06

File: cfgov/jinja2/v1/_includes/organisms/post-preview.html (hunk 3 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: Did in 555ef06

**Comment by jimmynotjim** [comment]
:+1:

**Comment by sebworks** [comment]
:+1:

**Pull request merged**